src/main/resources/application.properties

```diff
@@ -1,15 +1,22 @@
 server.port=8080
 
-spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
-spring.datasource.username=postgres
-spring.datasource.password=postgres
+spring.datasource.url=jdbc:postgresql://localhost:5432/patternpedia
+spring.datasource.username=patternpedia
+spring.datasource.password=patternpedia
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
 spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
 spring.jpa.hibernate.ddl-auto=update
 logging.level.com.patternpedia.api=debug
 spring.datasource.initialization-mode=always
 spring.jpa.properties.hibernate.enable_lazy_load_no_trans=true
+springdoc.swagger-ui.url=/patternpedia/v3/api-docs
+springdoc.swagger-ui.operationsSorter=alpha
+springdoc.default-produces-media-type=application/hal+json
+
 
 security.oauth2.resource.jwk.key-set-uri=http://localhost:8081/.well-known/jwks.json
 #okta.oauth2.issuer=https://dev-918271.okta.com/oauth2/default
-#okta.oauth2.clientId=0oa1eflyl1wZDVLLg357
\ No newline at end of file
+#okta.oauth2.clientId=0oa1eflyl1wZDVLLg357
+
+# Embedded Tomcat
+server.servlet.contextPath=/patternpedia
```

Comment:
should this change be reflected in the documentation (is a tomcat setup now needed/required)?

Reply:
The integrated Tomcat of Spring is still used. However, each path now contains this prefix. I added this to better differentiate between multiple backends in case qc-atlas and nisq-analyzer are running as well. They both also contain their respective contextPaths.